core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -63,12 +61,13 @@ enum CredentialAttrType
         private readonly string TeamIdKey = "teamIDHint";
 
         private string keychainGroup;
+        private RequestContext _requestContext;
```

Comment:
[NIT] I know we don't enforce coding style yet and that we sometimes use underscore for private members, but it would be good to be consistent with the MS coding style, i.e. class members should use camelCasing (not _usederscoreCamel and not PascalCasing)

Reply:
I agree...will fix.

Reply:
fun thing is that the middleware team likes the _ as it is what .netcore does...

---
In reply to: [216906861](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1251#discussion_r216906861) [](ancestors = 216906861)

Reply:
Leaving this as-is.